src/Microsoft.Fx.Portability.Reports.Html/Resources/LocalizedStrings.resx

```diff
@@ -153,9 +153,15 @@
   <data name="NotSupported" xml:space="preserve">
     <value>Not supported</value>
   </data>
+  <data name="PackageIdHeader" xml:space="preserve">
+    <value>PackageId</value>
+  </data>
   <data name="PortabilitySummaryPageTitle" xml:space="preserve">
     <value>Portability Summary</value>
   </data>
+  <data name="RecommendedChanges" xml:space="preserve">
+    <value>Recommended changes</value>
+  </data>
   <data name="RetargetingCompatIssueDescriptionPart1" xml:space="preserve">
     <value>Retargeting compatibility issues are breaking changes that only manifest when code is targeted to run on a newer .Net Framework version. This can happen if an application does not have a &lt;a href="https://msdn.microsoft.com/en-us/library/system.runtime.versioning.targetframeworkattribute%28v=vs.110%29.aspx"&gt;TargetFrameworkAttribute&lt;/a&gt; or when the application is rebuilt with a newer toolset.</value>
```

Comment:
The URL to TargetFrameworkAttribute should be: https://docs.microsoft.com/dotnet/api/system.runtime.versioning.targetframeworkattribute